csuntechlab/iSTART
# STR-275 -- Added toggle mutation for submit participants button

# Description
Added functionality to disable the submit button when pressed once.

# Testing
1. Ensure the submit button is hidden on submit.
2. Ensure they cannot call `submitParticipants` more than once via spamming.
